src/components/HeadFooter/HeadFooter.js

```diff
@@ -11,29 +11,29 @@ const overwrittenTheme = createMuiTheme({
 	palette: {
 		primary: {
 			main: '#C960FF',
-			light: '#d37fff',
-			dark: '#352A3A',
+			light: '#9C28C5',
+			dark: '#3E2961',
 			contrastText: '#fff'
 		},
 		secondary: {
-			main: '#ED3266',
-			light: '#f05b84',
-			dark: '#a52347',
+			main: '#FF477E',
+			light: '#f05b84', // did not receive updates for this color
```

Comment:
The comment won't make sense after a year. To convey some information to the reviewer, you can instead comment on your own PRs.